ramls/types/tags/tagUniqueCollectionItem.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Tag object schema for a collection",
+  "description": "Tag object schema for a collection of unique tags",
+  "javaType": "org.folio.rest.jaxrs.model.TagUniqueCollectionItem",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "UUID of tag assigned to the record",
```

Comment:
If unique tags don't really have any uuid associated with them, we should probably make it clear that this field will just contain a randomly generated uuid